BackEndApp/v1/views.py

```diff
@@ -494,3 +494,32 @@ def download_code(request):
     except Exception as e:
         response = HttpResponse("<h1>File not found</h1>")
     return response
+
+
+@api_view(["POST"])
+@is_authenticated
+def get_users(request):
+    try:
+        username = request.data.get("username")
+        user = User(username=username, password=None)
+        user = user.find()
+
+        users = os.listdir(os.path.expanduser("~") + "\.autodl")
```

Comment:
```suggestion
        users = os.listdir(user.rootpath)
```
Reason being if in the future we change the root path of the project from ~/.autodl to someplace else (Object/block storage on the internet) then we would need to make changes at multiple places :)

Reply:
I Made the requested changes, check it out.